manifests/params.pp

```diff
@@ -184,6 +183,12 @@
   $server_http_port           = 8139
   $server_http_allow          = []
 
+  # use puppetserver (JVM) or puppet master (Ruby)?
+  $server_implementation = $aio_package ? {
+    true    => 'puppetserver',
+    default => 'master',
+  }
+
```

Comment:
If this is just controlling implementation, rather than a whole host of stuff, that's fine. I can just override the implementation.